roles/ceph_prepare_installation/README.md

```diff
@@ -4,7 +4,7 @@ This role prepares the ceph installation by identifying if it's a new install an
 
 ## Requirements
 
-no requirement.
```

Comment:
same, why do you need this dependency ?

Reply:
Same as for `roles/add_livemigration_user`